packs/equipment/spellstriker-staff.json

```diff
@@ -21,7 +21,7 @@
+            "value": "<p>A <em>spellstriker staff</em> is wrought iron with gleaming arcane sigils etched into its surface and a sharp point at the bottom. Used as a weapon, the staff is a <em>+1 striking shifting staff</em>.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"action-glyph\">F</span> envision</p>\n<p><strong>Frequency</strong> once per hour</p>\n<p><strong>Trigger</strong> You use Spellstrike with a non-cantrip spell and miss with the Strike</p>\n<p><strong>Effect</strong> An explosion of magical energy—stored up for the Spellstrike—explodes out. All creatures in a @Template[type:emanation|distance:5] take 1d6 damage per rank of the spell, with a @Check[type:reflex|dc:resolve(@actor.attributes.spellDC.value)|basic:true] save against your spell DC. This damage is the same type the spell would have dealt, and the activation gains that trait; if the spell would have dealt multiple types of damage, choose one of them. If the damage of the explosion is mental, the save is @Check[type:will|dc:resolve(@actor.attributes.spellDC.value)|basic:true] instead of Reflex.</p>\n<hr />\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Effect</strong> You expend a number of charges from the staff to cast a spell from its list.</p>\n<ul>\n<li><strong>Cantrip</strong> @UUID[Compendium.pf2e.spells-srd.Item.Produce Flame]</li>\n<li><strong>1st</strong> @UUID[Compendium.pf2e.spells-srd.Item.Echoing Weapon], @UUID[Compendium.pf2e.spells-srd.Item.Sure Strike]</li>\n<li><strong>2nd</strong> @UUID[Compendium.pf2e.spells-srd.Item.Acid Arrow], @UUID[Compendium.pf2e.spells-srd.Item.Telekinetic Maneuver]</li>\n</ul>\n<hr />\n<p><strong>Craft Requirements</strong> You have the magus's Spellstrike activity. Supply one casting of all listed levels of all listed spells.</p>"
```

Comment:
```suggestion
            "value": "<p>A <em>spellstriker staff</em> is wrought iron with gleaming arcane sigils etched into its surface and a sharp point at the bottom. Used as a weapon, the staff is a <em>+1 striking shifting staff</em>.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"action-glyph\">F</span> envision</p>\n<p><strong>Frequency</strong> once per hour</p>\n<p><strong>Trigger</strong> You use Spellstrike with a non-cantrip spell and miss with the Strike</p>\n<p><strong>Effect</strong> An explosion of magical energy—stored up for the Spellstrike—explodes out. All creatures in a @Template[type:emanation|distance:5] take 1d6 damage per rank of the spell, with a @Check[type:reflex|dc:resolve(@actor.attributes.spellDC.value)|basic:true] save against your spell DC. This damage is the same type the spell would have dealt, and the activation gains that trait; if the spell would have dealt multiple types of damage, choose one of them. If the damage of the explosion is mental, the save is @Check[type:will|dc:resolve(@actor.attributes.spellDC.value)|basic:true] instead of Reflex.</p>\n<hr />\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Effect</strong> You expend a number of charges from the staff to cast a spell from its list.</p>\n<ul>\n<li><strong>Cantrip</strong> @UUID[Compendium.pf2e.spells-srd.Item.Produce Flame]</li>\n<li><strong>1st</strong> @UUID[Compendium.pf2e.spells-srd.Item.Echoing Weapon], @UUID[Compendium.pf2e.spells-srd.Item.Sure Strike]</li>\n<li><strong>2nd</strong> @UUID[Compendium.pf2e.spells-srd.Item.Acid Arrow], @UUID[Compendium.pf2e.spells-srd.Item.Telekinetic Maneuver]</li>\n</ul>\n<hr />\n<p><strong>Craft Requirements</strong> You have the magus's Spellstrike activity. Supply one casting of all listed ranks of all listed spells.</p>"
```